app/services/collators/capital_collator.rb

```diff
@@ -19,28 +15,19 @@ def call(submission_date:, capital_summary:, pensioner_capital_disregard:, maxim
                                                           smod_cap: maximum_subject_matter_of_dispute_disregard,
                                                           level_of_help:)
         property_smod = properties.sum(&:smod_allowance)
-        undisputed_vehicles = Assessors::VehicleAssessor.call(vehicles.reject(&:subject_matter_of_dispute), submission_date)
-        smod_vehicles = Assessors::VehicleAssessor.call(vehicles.select(&:subject_matter_of_dispute), submission_date)
-        vehicle_value = undisputed_vehicles.map(&:result).sum(&:assessed_value)
-        smod_vehicle_value = smod_vehicles.map(&:result).sum(&:assessed_value)
-        non_property_smod_allowance = Calculators::SubjectMatterOfDisputeDisregardCalculator.call(
-          disputed_capital_items: disputed_liquid + disputed_non_liquid,
-          disputed_vehicles: smod_vehicles.map(&:result),
-          maximum_disregard: maximum_subject_matter_of_dispute_disregard - property_smod,
-        )
+        assessed_vehicles = Assessors::VehicleAssessor.call(vehicles, submission_date)
 
         PersonCapitalSubtotals.new(
-          total_liquid: liquid_capital + smod_liquid_capital,
-          total_non_liquid: non_liquid_capital + smod_non_liquid_capital,
-          non_disputed_vehicles: undisputed_vehicles,
-          disputed_vehicles: smod_vehicles,
+          vehicles: assessed_vehicles,
           total_mortgage_allowance: property_maximum_mortgage_allowance_threshold(submission_date),
           pensioner_capital_disregard:,
-          disputed_non_property_disregard: non_property_smod_allowance,
           disputed_property_disregard: property_smod,
           properties:,
-          non_disputed_non_property_capital: liquid_capital + non_liquid_capital + vehicle_value,
-          disputed_non_property_capital: smod_liquid_capital + smod_non_liquid_capital + smod_vehicle_value - non_property_smod_allowance,
+          undisputed_nonliquid_capital: non_liquid_capital,
```

Comment:
@starswan What do you think of removing `undisputed_nonliquid_capital` and `disputed_nonliquid_capital` and pass `liquid_capital_items` like we did for vehicles?

Reply:
Yes my second commit did that for liquid and it seemed to work ok. Can do a 3rd one for non-liquid in the same vein